frontend/src/AddSeries/ImportSeries/SelectFolder/ImportSeriesSelectFolder.js

```diff
@@ -78,7 +78,10 @@ class ImportSeriesSelectFolder extends Component {
                       Make sure that your files include the quality in their filenames. eg. <span className={styles.code}>episode.s02e15.bluray.mkv</span>
                     </li>
                     <li className={styles.tip}>
-                      Point Sonarr to the folder containing all of your tv shows, not a specific one. eg. <span className={styles.code}>"{isWindows ? 'C:\\tv shows' : '/tv shows'}"</span> and not <span className={styles.code}>"{isWindows ? 'C:\\tv shows\\the simpsons' : '/tv shows/the simpsons'}"</span>
+                      Point Sonarr to the folder containing all of your tv shows, not a specific one. eg. <span className={styles.code}>"{isWindows ? 'C:\\tv shows' : '/tv shows'}"</span> and not <span className={styles.code}>"{isWindows ? 'C:\\tv shows\\the simpsons' : '/tv shows/the simpsons'}"</span> Additionally, each series must be in its own folder within the root/library folder.
+                    </li>
+                    <li className={styles.tip}>
+                      Do not use this Import method to import downloads from your download client. This is ONLY for existing organized libraries.
```

Comment:
`Import` should either be `Library Import` or possible reworded to `Do not use for importing downloads from your download client, this is only for existing organized libraries, not unsorted files.`

`ONLY` shouldn't be capitalized, we have better ways to highlight text if necessary, but I don't think it's actually needed here, people will either read it or they won't.